swaps/swaps_service.go

```diff
@@ -1036,7 +1045,7 @@ func (svc *swapsService) RefundSwap(swapId string) error {
 		logger.Logger.WithError(err).Error("Failed to fetch fee rate to create claim transaction")
 		return err
 	}
-	// TODO: generate a new key
+	// TODO: Use refundAddress
```

Comment:
You should _set_ the refund address here on the swap item in the DB. It still needs a new generated address